en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-310.md

```diff
@@ -2138,92 +2138,147 @@ Upgrade the WSO2 API Manager Analytics database from version 3.0.0 to version 3.
     ALTER TABLE APILASTACCESSSUMMARY ADD PRIMARY KEY (APINAME,APICREATOR,APIVERSION,APICREATORTENANTDOMAIN);
     ```
 !!! note
-    Type and name of a column of few tables were changed through WUM in version 2.6. It is important to add the above
-    change into your database prior to migration. So execute the below query to 
-    confirm whether the above change is already available in your system.
-    
-    ```
-    select AGG_COUNT from GEOLOCATIONAGG_SECONDS;
-    ```
-    If the above query returns an error indicating that the intended column can not be found, then please execute the 
-    below queries according to your db type to add the above change.
-    
-    ```tab="H2"
-    ALTER TABLE GeoLocationAgg_DAYS CHANGE totalCount AGG_COUNT bigint(20);
-    ALTER TABLE GeoLocationAgg_HOURS CHANGE totalCount AGG_COUNT bigint(20);
-    ALTER TABLE GeoLocationAgg_MINUTES CHANGE totalCount AGG_COUNT bigint(20);
-    ALTER TABLE GeoLocationAgg_MONTHS CHANGE totalCount AGG_COUNT bigint(20);
-    ALTER TABLE GeoLocationAgg_SECONDS CHANGE totalCount AGG_COUNT bigint(20);
-    ALTER TABLE GeoLocationAgg_YEARS CHANGE totalCount AGG_COUNT bigint(20);           
-    ```
+    Type and name of a column of few tables were changed through WUM in analytics version 2.6. It is important to 
+    add the above change into your database prior to migration. So execute the below queries which checks whether 
+    the above change is already available in your DB and if not it will add the relevant change. Ensure to replace 
+    "<Enter Analytics DB name here>" with the correct DB name in the scripts.   .
```

Comment:
Additional period (.)

Reply:
And is this "USE" statement required? We don't specify the "USE" statements usually anywhere? Instead, we specify, to which DB we need to run the queries. That has been already specified on line 2103

Reply:
It was required only in MSSQL. it expects to specify the intended Database beore running the stored procedures. For other db types its not used.